pkg/xtrabackup/xtrabackup.go

```diff
@@ -0,0 +1,377 @@
+package xtrabackup
+
+import (
+	"strconv"
+
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	apiv1alpha1 "github.com/percona/percona-server-mysql-operator/api/v1alpha1"
+	"github.com/percona/percona-server-mysql-operator/pkg/k8s"
+	"github.com/percona/percona-server-mysql-operator/pkg/util"
+	"github.com/pkg/errors"
+)
+
+const (
+	componentName      = "xtrabackup"
+	componentShortName = "xb"
+	dataVolumeName     = "datadir"
+	dataMountPath      = "/var/lib/mysql"
+	credsVolumeName    = "users"
+	credsMountPath     = "/etc/mysql/mysql-users-secret"
+	tlsVolumeName      = "tls"
+	tlsMountPath       = "/etc/mysql/mysql-tls-secret"
+	backupVolumeName   = componentName
+	backupMountPath    = "/backup"
+)
+
+func Name(cluster *apiv1alpha1.PerconaServerMySQL) string {
+	return cluster.Name + "-" + componentShortName
```

Comment:
I want to see backup storage name  in backup pod name . As in pxc operator. 

`xb-on-demand-backup-aws-s3-wv9kr`

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/77/commits/d6ca4bf746d8f477bb6c535d200470c5b4325aea